### ISSUE TITLE
Training and cpu inferencing

### ISSUE THREAD

**Issue by seekingdeep** [Leedeng/SauvolaNet#2]
@Leedeng Hi there,

- is sauvolanet able to inference using cpu-only
- how to train sauvolanet

**Comment by mohamadmansourX** [comment]
Hello,
> how to train sauvolanet

I was able to build some wrappers and easy to use [Configuration File](https://github.com/mohamadmansourX/SauvolaNet-Training/blob/master/docs/Training_Config.md) for training the model.
Just play with the [Config file](https://github.com/mohamadmansourX/SauvolaNet-Training/blob/master/docs/Training_Config.md) and run `$python train.py` and you're done.
Hope I can merge it with this Github later!
https://github.com/mohamadmansourx/SauvolaNet-Training

> is sauvolanet able to inference using cpu-only

1- If you used my wrappers:
This can be done by simply setting [use_gpu](https://github.com/mohamadmansourX/SauvolaNet-Training/blob/master/docs/Training_Config.md#global) to False in the Config.yaml

2- If not:
You're trying to use Keras on CPU ([issue](https://github.com/keras-team/keras/issues/4613#issue-193596034)). 
I believe the easiest way is to hide cuda devices before importing Tensorflow.
In other words, **at the very beginning of your py file** just write the below:
```python
import os
os.environ['CUDA_VISIBLE_DEVICES']='-1' #This will hide Cuda devices from TF (incase you have GPU on your pc)
import tensorflow #tensorflow will now be running on your CPU only and you can load the model normally
```

**Comment by seekingdeep** [comment]
@mohamadmansourX it seems that you have done an amazing job, thanks!!
